### DISCUSSION TITLE
計中Azure學生研究方案

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS3073-C/Python-Community-GS3073-C#484] [General]
如果大家有研究(如:專題)需求，可以申請電算中心的Azure學生研究方案(需有指導教授簽名)，相較於老師在課堂上介紹的Azure學生版的100美金額度，計中提供的學生研究方案有14000台幣左右的額度可以使用，在資源使用上的限制也會比較少。

https://www.cc.ncu.edu.tw/page/azure

**Comment by lx3211** [comment]
謝謝分享，感覺是一個資源對專題很有幫助

**Comment by williamyen0623** [comment]
中大計中我的超人，讓學生不用煩惱研究經費的問題

**Comment by limitx0** [comment]
原來計中有這麼多資源，太讚了
以後做研究可以去申請看看

**Comment by Yilin20021016** [comment]
原Azure還有這樣的方案 感覺對我未來很有用 感謝

**Comment by Raywei0214** [comment]
謝謝提供資訊，我會善加利用雖然目前還用不到

**Comment by awkward-willy** [comment]
很開心能知道這樣的資訊，讓學生能不用花大錢就體驗到雲端平台的美好，這樣就不用再各大平台用學生身分寄生了，也不用因為免費的平台關閉特定方案而煩惱，是學生族群的一大福音！
